diff -r de2e5a9fee0a src/base/string.cpp
--- a/src/base/string.cpp	Sat Mar 17 20:50:25 2018 +0000
+++ b/src/base/string.cpp	Mon Nov 05 22:03:21 2018 +0100
@@ -710,7 +710,7 @@
 
 
 #if defined HAVE_WINDOWS_H
-#include <Windows.h> // for WideCharToMultiByte
+#include <windows.h> // for WideCharToMultiByte
 #endif
 
 void cc_string_set_wtext(cc_string * me, const wchar_t * text)
diff -r de2e5a9fee0a src/shapenodes/SoFaceSet.cpp
--- a/src/shapenodes/SoFaceSet.cpp	Sat Mar 17 20:50:25 2018 +0000
+++ b/src/shapenodes/SoFaceSet.cpp	Mon Nov 05 22:03:21 2018 +0100
@@ -568,7 +568,7 @@
     }
 
     // check if we can render things using glDrawArrays
-    if (SoGLDriverDatabase::isSupported(sogl_glue_instance(state), SO_GL_VERTEX_ARRAY) &&
+/*    if (SoGLDriverDatabase::isSupported(sogl_glue_instance(state), SO_GL_VERTEX_ARRAY) &&
         (PRIVATE(this)->primitivetype == GL_TRIANGLES) ||
         (PRIVATE(this)->primitivetype == GL_QUADS) &&
         (nbind != PER_FACE) &&
@@ -588,7 +588,7 @@
                               doTextures, (mbind == PER_VERTEX));
       
     }
-    else {
+    else*/ {
       SOGL_FACESET_GLRENDER(nbind, mbind, doTextures, (coords,
                                                        normals,
                                                        &mb,
diff -r de2e5a9fee0a src/threads/thread.cpp
--- a/src/threads/thread.cpp	Sat Mar 17 20:50:25 2018 +0000
+++ b/src/threads/thread.cpp	Mon Nov 05 22:03:21 2018 +0100
@@ -83,6 +83,10 @@
 #include "threads/syncp.h"
 #include "tidbitsp.h"
 
+#if defined HAVE_WINDOWS_H
+#include <windows.h> // for WideCharToMultiByte
+#endif
+
 
 /* ********************************************************************** */
 
