diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 64d5e87..bf2da9e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -622,7 +622,7 @@ ENDIF (MSVC)
 SET (CMD $<TARGET_FILE:H5detect>)
 ADD_CUSTOM_COMMAND (
     OUTPUT ${HDF5_BINARY_DIR}/H5Tinit.c
-    COMMAND ${CMD}
+    COMMAND wine ${CMD}
     ARGS > ${HDF5_BINARY_DIR}/H5Tinit.c
     DEPENDS H5detect
 )
@@ -636,7 +636,7 @@ ENDIF (MSVC)
 SET (CMD $<TARGET_FILE:H5make_libsettings>)
 ADD_CUSTOM_COMMAND (
     OUTPUT ${HDF5_BINARY_DIR}/H5lib_settings.c
-    COMMAND ${CMD}
+    COMMAND wine ${CMD}
     ARGS > ${HDF5_BINARY_DIR}/H5lib_settings.c
     DEPENDS H5make_libsettings
     WORKING_DIRECTORY ${HDF5_BINARY_DIR}
