--- /usr/x86_64-w64-mingw32/sys-root/mingw/include/time.h	2018-11-03 12:00:01.432637120 +0000
+++ /usr/x86_64-w64-mingw32/sys-root/mingw/include/time.h	2018-11-03 12:26:04.942880530 +0000
@@ -266,12 +266,18 @@
 #endif
 
 #ifdef _POSIX_THREAD_SAFE_FUNCTIONS
+#ifdef __cplusplus
+extern "C" {
+#endif
 __forceinline struct tm *__cdecl localtime_r(const time_t *_Time, struct tm *_Tm) {
   return localtime_s(_Tm, _Time) ? NULL : _Tm;
 }
 __forceinline struct tm *__cdecl gmtime_r(const time_t *_Time, struct tm *_Tm) {
   return gmtime_s(_Tm, _Time) ? NULL : _Tm;
 }
+#ifdef __cplusplus
+}
+#endif
 __forceinline char *__cdecl ctime_r(const time_t *_Time, char *_Str) {
   return ctime_s(_Str, 0x7fffffff, _Time) ? NULL : _Str;
 }
--- /usr/x86_64-w64-mingw32/sys-root/mingw/include/c++/ctime	2018-11-03 12:00:18.046358864 +0000
+++ /usr/x86_64-w64-mingw32/sys-root/mingw/include/c++/ctime	2018-11-03 11:38:45.000000000 +0000
@@ -51,8 +51,8 @@
 #undef time
 #undef asctime
 #undef ctime
-#undef gmtime
-#undef localtime
+//#undef gmtime
+//#undef localtime
 #undef strftime
 
 namespace std
