--- qtwebkit-opensource-src-5.5.1/Tools/qmake/mkspecs/features/features.prf	2018-11-15 14:46:06.631067204 +0100
+++ qtwebkit-opensource-src-5.5.1/Tools/qmake/mkspecs/features/features.prf	2018-11-15 14:46:28.610141512 +0100
@@ -83,10 +83,6 @@
         WEBKIT_CONFIG += plugin_architecture_unsupported
     }
 
-    # Enable the USE(3D_GRAPHICS) flag when QtOpenGL is enabled.
-    # Disable on Windows CE for now, as ANGLE won't compile.
-    !wince*:contains(QT_CONFIG, opengl): WEBKIT_CONFIG += use_3d_graphics
-
     # Temporarily disable FTPDIR on Windows CE (missing functions from time.h)
     wince* {
         WEBKIT_CONFIG -= ftpdir
