diff -rupN Python-3.7.4/Modules/_ctypes/_ctypes.c Python-3.7.4_new/Modules/_ctypes/_ctypes.c
--- Python-3.7.4/Modules/_ctypes/_ctypes.c	2019-07-08 18:03:50.000000000 +0000
+++ Python-3.7.4_new/Modules/_ctypes/_ctypes.c	2020-02-29 17:46:03.466144329 +0000
@@ -5588,16 +5588,12 @@ PyInit__ctypes(void)
 #endif
 
 /* If RTLD_LOCAL is not defined (Windows!), set it to zero. */
-#if !HAVE_DECL_RTLD_LOCAL
 #define RTLD_LOCAL 0
-#endif
 
 /* If RTLD_GLOBAL is not defined (cygwin), set it to the same value as
    RTLD_LOCAL.
 */
-#if !HAVE_DECL_RTLD_GLOBAL
 #define RTLD_GLOBAL RTLD_LOCAL
-#endif
 
     PyModule_AddObject(m, "RTLD_LOCAL", PyLong_FromLong(RTLD_LOCAL));
     PyModule_AddObject(m, "RTLD_GLOBAL", PyLong_FromLong(RTLD_GLOBAL));
