--- mingw-python3.spec	2020-02-29 17:53:20.329301195 +0000
+++ mingw-python3.spec_new	2020-02-29 17:53:08.794191307 +0000
@@ -150,6 +150,7 @@ Patch94:       2030-fix-msvc9-import.pat
 # Fedora specific patches
 # Get windres executable name from environment
 Patch100:      %{name}_windres.patch
+Patch101:      python-ctypes.patch
 
 
 BuildRequires: automake autoconf libtool
@@ -305,6 +306,7 @@ rm -f Misc/config_mingw \
 %patch93 -p1
 %patch94 -p1
 %patch100 -p1
+%patch101 -p1
 
 autoreconf -vfi
 
