--- octave/octrun.swg	2015-08-02 23:51:59.000000000 +0200
+++ octave/octrun.swg	2015-12-28 19:13:40.000000000 +0100
@@ -528,10 +528,10 @@
 	delete this;
     }
 
-    long swig_this() const {
+    size_t swig_this() const {
       if (!types.size())
-	return (long) this;
-      return (long) types[0].second.ptr;
+	return (size_t) this;
+      return (size_t) types[0].second.ptr;
     }
     const char* help_text() const {
       if (!types.size())
