--- src/Makefile.in	2018-11-03 18:00:00.144539918 +0000
+++ src/Makefile.in	2018-11-03 18:00:25.072543327 +0000
@@ -854,7 +854,7 @@
 am__v_lt_0 = --silent
 am__v_lt_1 = 
 libxerces_c_la_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CXX \
-	$(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=link $(CXXLD) \
+	$(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=link $(CXXLD) -no-undefined \
 	$(AM_CXXFLAGS) $(CXXFLAGS) $(libxerces_c_la_LDFLAGS) \
 	$(LDFLAGS) -o $@
 AM_V_P = $(am__v_P_@AM_V@)
--- src/xercesc/util/KVStringPair.hpp	2018-11-03 22:00:14.670852344 +0100
+++ src/xercesc/util/KVStringPair.hpp	2018-11-03 22:17:45.520889248 +0100
@@ -71,33 +71,33 @@
     //
     //  We support the
     // -----------------------------------------------------------------------
-    const XMLCh* getKey() const;
-    XMLCh* getKey();
-    const XMLCh* getValue() const;
-    XMLCh* getValue();
+    inline const XMLCh* getKey() const;
+    inline XMLCh* getKey();
+    inline const XMLCh* getValue() const;
+    inline XMLCh* getValue();
 
 
     // -----------------------------------------------------------------------
     //  Setters
     // -----------------------------------------------------------------------
-    void setKey(const XMLCh* const newKey);
-    void setValue(const XMLCh* const newValue);
-    void setKey
+    inline void setKey(const XMLCh* const newKey);
+    inline void setValue(const XMLCh* const newValue);
+    inline void setKey
     (
         const   XMLCh* const newKey
         , const XMLSize_t    newKeyLength
     );
-    void setValue
+    inline void setValue
     (
         const   XMLCh* const newValue
         , const XMLSize_t    newValueLength
     );
-    void set
+    inline void set
     (
         const   XMLCh* const newKey
         , const XMLCh* const newValue
     );
-    void set
+    inline void set
     (
         const     XMLCh* const newKey
         , const   XMLSize_t    newKeyLength
